versions/draft-3/SPEC.md

```diff
@@ -3053,6 +3055,30 @@ Array[String] zs = [ "d", "e" ]
 Array[Pair[Int, String]] crossed = cross(xs, zs) # i.e. crossed = [ (1, "d"), (1, "e"), (2, "d"), (2, "e"), (3, "d"), (3, "e") ]
 ```
 
+## Array[Pair[X,Y]] as_pairs(Map[X,Y])
+
+Given any two Object types in a Map, the `as_pairs` function returns an Array containing each element in the form of a Pair. The key will be the left element of the Pair and the value the right element.
```

Comment:
I don't like `Object` here since that is an existing, concrete WDL type (at least for now). I also don't think it's needed here - how about `"Given a Map[X, Y], return an Array[Pair[X, Y]] containing ..."`